data/brands/shop/computer.json

```diff
@@ -198,6 +198,34 @@
         "name:zh-Hant": "偉倫電腦",
         "shop": "computer"
       }
+    },
+    {
+      "displayName": "ドスパラ",
+      "id": "dospara-8cc09d",
+      "locationSet": {"include": ["jp"]},
+      "tags": {
+        "brand": "ドスパラ",
+        "brand:en": "Dospara",
+        "brand:ja": "ドスパラ",
+        "brand:wikidata": "Q11306200",
+        "name": "ドスパラ",
+        "name:en": "Dospara",
+        "name:ja": "ドスパラ",
+        "shop": "computer"
+      }
+    },
+    {
+      "displayName": "パソコン工房",
+      "id": "6185e9-8cc09d",
+      "locationSet": {"include": ["jp"]},
+      "tags": {
+        "brand": "パソコン工房",
+        "brand:ja": "パソコン工房",
```

Comment:
With brand:en/name:en=UNITCOM?

Reply:
UNITCOM is the name of the company, and since I could not find an official English name for "パソコン工房( In Romaji: Pasokon Kōbō )", I decided to leave the English name out for now.